.github/workflows/build-test-publish.yml

```diff
@@ -184,6 +184,11 @@ jobs:
         datalad update -r --merge -d hcph-pilot_fieldmaps/
         datalad get -r -J 2 -d hcph-pilot_fieldmaps/ hcph-pilot_fieldmaps/*
 
+        # ds005250
+        datalad install -r https://gin.g-node.org/tsalo/ds005250-sdcflows.git
+        datalad update -r --merge -d ds005250-sdcflows/
+        datalad get -r -J 2 -d ds005250-sdcflows/ ds005250-sdcflows/sub-04/ses-2/
```

Comment:
I set up ses-2 with IntendedFor and ses-1 with B0Field fields. The B0Fields aren't working.

Reply:
Also, it looks like `BIDSLayout.get_B0FieldIdentifiers` doesn't work at all.

Reply:
are your B0Field* lists? see https://github.com/bids-standard/pybids/pull/684

Reply:
They are! I didn't realize that was a known issue. I can modify my test dataset to just have strings for B0FieldIdentifier.